packages/mui-material/src/styles/CssVarsProvider.js

```diff
@@ -0,0 +1,35 @@
+import { unstable_createCssVarsProvider as createCssVarsProvider } from '@mui/system';
+import { defaultOpacity } from './unstable_createTheme';
+import createTheme from './createTheme';
```

Comment:
Why this is used as a default theme instead of `unstable_createTheme`?

If I do this:
```js
<CssVarsProvider>
  <Button>...</Button>
</CssVarsProvider>
```
Would the `channel` tokens be generated?

Reply:
Oops, I've created the `unstable_createTheme` afterwards, and forgot to update :) You are correct we need this.

Reply:
Done